tests/unit/s2n_server_key_share_extension_test.c

```diff
@@ -488,221 +518,429 @@ int main(int argc, char **argv)
                 .ecc_preferences = &s2n_ecc_preferences_20200310,
         };
 
-        EXPECT_SUCCESS(s2n_enable_tls13());
+        const struct s2n_kem_group *kem_groups_sike_bike[] = {
+                &s2n_secp256r1_sike_p434_r2,
+                &s2n_secp256r1_bike1_l1_r2
+        };
+
+        const struct s2n_kem_preferences kem_prefs_sike_bike = {
+                .kem_count = 0,
+                .kems = NULL,
+                .tls13_kem_group_count = s2n_array_len(kem_groups_sike_bike),
+                .tls13_kem_groups = kem_groups_sike_bike,
+        };
+
+        const struct s2n_security_policy security_policy_sike_bike = {
+                .minimum_protocol_version = S2N_SSLv3,
+                .cipher_preferences = &cipher_preferences_test_all_tls13,
+                .kem_preferences = &kem_prefs_sike_bike,
+                .signature_preferences = &s2n_signature_preferences_20200207,
+                .ecc_preferences = &s2n_ecc_preferences_20200310,
+        };
+
+        /* Tests for s2n_server_key_share_extension.recv with hybrid PQ key shares */
```

Comment:
Note: tests from lines 541 to 760 are previously committed tests and were not changed here (I think the git diff is detecting indentation changes).